dev-server/static/scripts/index.js

```diff
@@ -22,4 +22,15 @@
       toggleClientButton.textContent = 'Unload client';
     }
   };
+
+  // Setup document links whose URLs have a randomly generated suffix parameter.
```

Comment:
Pedantry: _Set up_

Perhaps: _Update `href` attributes for `a.js-randomize-url` elements to add a randomly-generated suffix parameter_

or similar?

I note that the code acts on the elements assuming an `href` attribute is valid for them, i.e. perhaps the selector could be more specific (`a.js-randomize-url`)?

Reply:
> Pedantry: Set up

Indeed! 😔 

> Perhaps: Update href attributes for a.js-randomize-url elements to add a randomly-generated suffix parameter

Mmm ... I think this might be repeating the code a little too much.

> I note that the code acts on the elements assuming an href attribute is valid for them, i.e. perhaps the selector could be more specific (a.js-randomize-url)?

We're trusting the developer to only use this on things that have an `href` for now, which is OK since we control all the content. By not adding the `a` filter the dev will at least get a console error if they try to use this class on an element where it isn't supported rather than it silently not working.